modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractCppCodegen.java

```diff
@@ -191,7 +197,7 @@ public String toVarName(String name) {
             return escapeReservedWord(name);
         }
 
-        if (name.length() > 1) {
+        if (variableNameFirstCharacterUppercase && name.length() > 1) {
             return sanitizeName(Character.toUpperCase(name.charAt(0)) + name.substring(1));
```

Comment:
Sanitizing is needed even if the variable name first character is not upper case.
i.e. **v1.varname** and  **V1.varname**

Reply:
I agree, that's why this exists
https://github.com/OpenAPITools/openapi-generator/blob/6c0f9c7f2b36847e38110d4eda4a2d28754832c0/modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractCppCodegen.java#L204

Reply:
It will get sanitized both ways. Line 204 is hit if condition fails